common/enablers/enableRoundsHandler.go

```diff
@@ -63,16 +65,109 @@ func getRoundConfig(args config.RoundConfig, configName string) (*roundFlag, err
 		"options", fmt.Sprintf("[%s]", strings.Join(activationRound.Options, ", ")))
 
 	return &roundFlag{
-		Flag:    &atomic.Flag{},
 		round:   round,
 		options: activationRound.Options,
 	}, nil
 }
 
+func (handler *enableRoundsHandler) createAllFlagsMap(conf config.RoundConfig) error {
+	disableAsyncCallV1, err := getRoundConfig(conf, string(common.DisableAsyncCallV1Flag))
+	if err != nil {
+		return err
+	}
+
+	supernovaEnableRound, err := getRoundConfig(conf, string(common.SupernovaRoundFlag))
+	if err != nil {
+		return err
+	}
+
```

Comment:
Maybe specific, precise error messages (can be inline)?

Reply:
added